content/en/docs/_index.md

```diff
@@ -2,3 +2,9 @@
 title = "Documentation"
 description = "All of Kubeflow documentation"
 +++
+
+{{% alert title="Out of date" color="warning" %}}
```

Comment:
_index.md files mainly contain content from other pages. Would it be possible to exclude the _index.md pages from this PR?

Reply:
Hi @joeliedtke, sure!